apps/sms/js/navigation.js

```diff
@@ -116,6 +116,17 @@ var Navigation = {
   },
 
   /**
+   * Ensures that current panel is correctly set, if no current panel set and
+   * we're not in the process of transitioning then set default panel as current
+   * one.
+   */
+  ensureCurrentPanel: function() {
```

Comment:
Also was thinking to return a promise here, smth like this:
```
if (this.transitioning) {
  return xPromise;
}

if (!currentPanel) {
  return this.toPanel(this.defaultPanel);
}

return Promise.resolve();
```

But don't have anything for ```xPromise```, that's how the following question appeared: do you know why can't we use smth like ```Navigation.transitionPromise (that equals to local promise from toPanel)``` instead of boolean ```Navigation.transitioning```?

It's not important as there is no need in this promise currently :)